ledger-core/virtual/integration/delegated_finished_test.go

```diff
@@ -73,234 +63,3 @@ func mockExecutor(t *testing.T, server *utils.Server, callMethod callMethodFunc,
-	}
-}
-
-// In this case VDelegatedRequestFinished must not contain new state since it was not changed
-func TestVirtual_SendDelegatedFinished_IfPulseChanged_Without_SideEffect(t *testing.T) {
-	t.Log("C4990")
-
-	var (
-		mc = minimock.NewController(t)
-	)
-
-	server, ctx := utils.NewUninitializedServer(nil, t)
-	defer server.Stop()
-
-	executeDone := server.Journal.WaitStopOf(&execute.SMExecute{}, 1)
-
-	runnerMock := logicless.NewServiceMock(ctx, mc, nil)
-	server.ReplaceRunner(runnerMock)
-	server.Init(ctx)
-	server.IncrementPulseAndWaitIdle(ctx)
-
-	var (
-		class        = testwallet.GetClass()
-		objectLocal  = server.RandomLocalWithPulse()
-		objectGlobal = reference.NewSelf(objectLocal)
-
-		typedChecker = server.PublisherMock.SetTypedChecker(ctx, mc, server)
-	)
-
-	{
-		typedChecker.VCallResult.SetResend(false)
-		typedChecker.VDelegatedCallResponse.SetResend(true)
-		typedChecker.VDelegatedCallRequest.SetResend(true)
-		typedChecker.VStateReport.SetResend(true)
-		typedChecker.VDelegatedRequestFinished.Set(func(finished *payload.VDelegatedRequestFinished) bool {
-			require.Equal(t, objectGlobal, finished.Callee)
-			require.Equal(t, payload.CTMethod, finished.CallType)
-			require.Equal(t, payload.BuildCallFlags(contract.CallTolerable, contract.CallDirty), finished.CallFlags)
-
-			require.Nil(t, finished.LatestState)
-
-			return true
-		})
-	}
-
-	Method_PrepareObject(ctx, server, payload.Ready, objectGlobal)
-
-	{
-		pl := payload.VCallRequest{
-			CallType:            payload.CTMethod,
-			CallFlags:           payload.BuildCallFlags(contract.CallTolerable, contract.CallDirty),
-			Caller:              server.GlobalCaller(),
-			Callee:              objectGlobal,
-			CallSiteDeclaration: class,
-			CallSiteMethod:      "AddAmount",
-			CallOutgoing:        server.BuildRandomOutgoingWithPulse(),
-			Arguments:           insolar.MustSerialize([]interface{}{}),
-		}
-
-		key := pl.CallOutgoing.String()
-		runnerMock.AddExecutionMock(key).
-			AddStart(func(execution execution.Context) {
-				server.IncrementPulse(ctx)
-			}, &execution.Update{
-				Type:   execution.Done,
-				Result: requestresult.New(makeEmptyResult(), objectGlobal),
-			})
-		runnerMock.AddExecutionClassify(key, contract.MethodIsolation{
-			Interference: contract.CallTolerable,
-			State:        contract.CallDirty,
-		}, nil)
-
-		server.SendPayload(ctx, &pl)
-	}
-
-	testutils.WaitSignalsTimed(t, 10*time.Second, executeDone)
-	testutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
-
-	{
-		require.Equal(t, typedChecker.VCallResult.Count(), 1)
-		require.Equal(t, typedChecker.VDelegatedCallResponse.Count(), 1)
-		require.Equal(t, typedChecker.VDelegatedCallRequest.Count(), 1)
-		require.Equal(t, typedChecker.VStateReport.Count(), 1)
-		require.Equal(t, typedChecker.VDelegatedRequestFinished.Count(), 1)
-	}
-}
-
-func TestVirtual_SendDelegatedFinished_IfPulseChanged_Constructor(t *testing.T) {
-	t.Log("C4988")
```

Comment:
There is a better test with the same scenario: `TestVirtual_Constructor_PulseChangedWhileOutgoing`, caseID = "C5085"